helm/cluster/values.schema.json

```diff
@@ -1186,6 +1189,24 @@
                                 },
                                 "preKubeadmCommands": {
                                     "$ref": "#/$defs/preKubeadmCommands"
+                                },
+                                "rolloutBefore": {
+                                    "type": "object",
+                                    "title": "Rollout Before",
+                                    "description": "Rollout Before is a field to indicate a rollout should be performed if the specified criteria is met.",
+                                    "required": [
+                                        "certificatesExpiryDays"
+                                    ],
+                                    "properties": {
+                                        "certificatesExpiryDays": {
+                                            "type": "integer",
+                                            "title": "Certificate expiry days",
+                                            "description": "Indicates a rollout needs to be performed if the certificates of the machine will expire within the specified days.",
+                                            "default": 180,
+                                            "maximum": 364,
+                                            "minimum": 30
```

Comment:
CAPI docs recommends to set a long enough period of time before the certificates expire, so that there is enough time for the rotation to complete. 1 month should be enough time.